src/StatusQ/Components/StatusListItem.qml

```diff
@@ -21,8 +21,10 @@ Rectangle {
     property bool enabled: true
     property bool highlighted: false
     property bool propagateTitleClicks: true 
+    property bool clickable: true
```

Comment:
We can remove this. You get access to `sensor` which can be `enabled`. So within and outside of this component you can check for `sensor.enabled`

Reply:
@PascalPrecht  even when I disable the sensor, there is still one part which have a mouse area enable:

https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Components/StatusListItem.qml#L175-L184

Do you think this is a bug?

Reply:
synced offline, this was a bug and it should be fixed now